src/test/java/com/training/web/tests/flipkart/PaymentPageTests.java

```diff
@@ -38,4 +38,16 @@ public void testEmiOptions(String bankName) {
         softAssertions.assertThat(paymentPage.getEmiSupport(bankName)).as("Bank details inaccurate")
                 .isEqualTo("No");
     }
+
+    @Test
+    public void testBackToTopFunctionality(){
+
+        paymentPage.scrollToPageEnd().clickBackToTop();
```

Comment:
after scrolling to pageEnd, add an assertion to verify if backToTop is displayed -> as true